lib/src/Base/Graph/openturns/DrawableImplementation.hxx

```diff
@@ -300,6 +303,9 @@ protected:
   /** Line width of the curve */
   Scalar lineWidth_;
 
+  /** Explicit color validation flag */
+  Bool isColorExplicitlySet_;
```

Comment:
```suggestion
  Bool isColorExplicitlySet_ = false;
```
It can be initialized here in the declaration instead of the 2 ctors's initialization list